### DISCUSSION TITLE
Repository with some nice examples

### DISCUSSION THREAD

**Discussion by HenkVanMaanen** [mysticaltech/terraform-hcloud-kube-hetzner#146] [Ideas]
This repository: https://github.com/xunleii/terraform-module-k3s

Has some nice examples for things like draining nodes on destroy, addind taints and labels without recreating nodes etc.

**Comment by mysticaltech** [comment]
@HenkVanMaanen Thanks for sharing! Worth extracting the good stuff! If you feel like doing a PR, please shoot!
